src/mrsimulator/signal_processing/baseline.py

```diff
@@ -22,8 +22,8 @@ def function(self):
         return "baseline"
 
 
-class Polynomial(Baseline):
-    r"""Add a baseline polynomial to all dependent variables (y) in the CSDM object.
+class Trinomial(Baseline):
+    r"""Add a baseline Trinomial to all dependent variables (y) in the CSDM object.
```

Comment:
Now that there is a generic polynomial baseline function class, is the trinomial baseline function needed?